python/tests/parsers/test_parser_utils.py

```diff
@@ -0,0 +1,363 @@
+                                       voting_authority_shared=0,
+                                       voting_authority_none=1095)]
+        self.data_13f2_row2 = [Data13f(equity_holdings_id='8g586856b668j6',
+                                       accession_no=ACCESSION_NO_TABLE1_ROW2,
+                                       cik_no=COMPANY_CIK_2,
+                                       name_of_issuer='ALPS ETF TR',
+                                       title_of_class='SECTR DIV DOGS',
+                                       cusip='00162Q858',
+                                       value=2229,
+                                       ssh_prnamt=48594,
+                                       ssh_prnamt_type='SH',
+                                       put_call='None',
+                                       investment_discretion='SOLE',
+                                       other_manager='None',
+                                       voting_authority_sole=0,
+                                       voting_authority_shared=0,
+                                       voting_authority_none=48594)]
+
+    def tearDown(self):
+        """Tears down test database"""
+        db.session.remove()
+        db.drop_all()
+
+
+def test_create_url_list():
+    """Test for create_url_list"""
+
+    actual = create_url_list(DATE_1)
+
+    assert actual == [
+        'https://www.sec.gov/Archives/edgar/data/1478997/0001478997-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/819864/0000819864-21-000002-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1567784/0000909012-21-000002-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1479844/0001479844-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1362987/0001362987-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1542265/0001542265-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/740272/0000740272-21-000002-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1799284/0001799284-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1800336/0001800336-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1744348/0001754960-21-000002-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1664017/0001664017-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1370102/0001370102-21-000003-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1766067/0001214659-21-000321-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1008937/0001008937-21-000001-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1761450/0001761450-21-000004-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1015308/0001015308-21-000002-index.html',
+        'https://www.sec.gov/Archives/edgar/data/1387399/0001567619-21-000762-index.html']
+
+
+def filing_detail_13f_text_1():
```

Comment:
its kind of weird that these are down here. Feels like they should be up with the fixtures or all the way at the bottom.

Reply:
wait... are they supposed to be fixtures?